Core/clim-core/standard-presentations.lisp

```diff
@@ -682,8 +681,10 @@
         (t (getf (cdr element) :value))))
 
 (defun member-alist-doc-key (element)
-  (if (and (consp element) (consp (cdr element)) (consp (cddr element)))
-      (getf (cdr element) :documentation)))
+  (when (and (consp element)
+             (consp (cdr element))
+             (consp (cddr element)))
```

Comment:
How about `(typep element '(cons t (cons t cons)))`? (Not that I understand what this check does)

Reply:
this was just stylistic line break change and when->if. a function above also uses cddr etc, so having it this way is more consistent.